operating/external-ips.adoc

```diff
@@ -0,0 +1,216 @@
+[id="external-ips"]
+= External IPs
```

Comment:
below this add:
include::modules/common-attributes.adoc[]

Reply:
Done